conf/application.conf

```diff
@@ -95,6 +95,10 @@ microservice {
   }
 }
 
+features {
+  coreVatReturns = true
```

Comment:
Just a reminder that we add the config for false for the other envs. 👍🏻

Reply:
I have it set in app-config-base as false, does that override this?